src/GraphQL.Federation/GraphQL.Federation.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.0</TargetFrameworks>
```

Comment:
netcoreapp3.0 has end of life on March 3, 2020, please change TFM

Reply:
I will convert this into NET standard library once i move HttpRequestExtension out.